net/fetcher_test.go

```diff
@@ -36,6 +37,15 @@ func TestFetchHappyPath(t *testing.T) {
 	block2 := builder.NewGenesis().At(0)
 	block3 := builder.NewGenesis().At(0)
 
+	// TODO this goes away in this PR after moving to graphsync fetcher
```

Comment:
This TODO needs an issue URL.

Reply:
As with the other one this TODO and code are going away before this PR merges.